IFC4x3/PropertyConstants/y/YES_0YdhpMIHj3Cxi9PWYJj7S9.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="YES_0YdhpMIHj3Cxi9PWYJj7S9" Name="YES" UniqueId="229ebcd6-491b-4333-bb09-660893b47709" />
```

Comment:
`IfcBoolean`?

Reply:
WIP -> Question To Claude, Andreas and domain expert.

Reply:
Done. Commit : e5e747f8ce6e67ed1e5301b5fb1fba218df02d67